robotframework_reportportal/listener.py

```diff
@@ -234,3 +233,16 @@ def end_keyword(self, _, attributes, ts=None):
         kwd = self._finish_current_item().update(attributes)
         logger.debug('ReportPortal - End Keyword: {0}'.format(kwd.attributes))
         self.service.finish_keyword(keyword=kwd, ts=ts)
+
+    def report_file(self, report_path):
+        """Attach HTML report created by Robot Framework to RP launch.
+
+        :param report_path: Path to the report file
+        """
+        if self.variables.attach_report:
+            message = {'message': 'Execution report', 'level': 'INFO'}
+            self.log_message_with_image(message, report_path)
```

Comment:
In that case you will attach just a hypertext information, but it may contain images (like in case of "take screenshot as file" keyword).
I see 2 options here:
* Scan for images and other embeddings, pack it into web-archive and attach them too.
* Leave it as is but open an issue to attach images and other embeddings for future.

Reply:
RF HTML report file is a self-sufficient file and it does not require packaging. Screenshots are not included in that report.

Reply:
OK